ui/cypress-ci/status/index.js

```diff
@@ -1,16 +1,16 @@
 import React from "react";
 
-function Index({ commitId, result, status }) {
+function Index({ commitId, result, status , commitBuildHref}) {
   const success = [2, 4].indexOf(result) !== -1 ? true : false;
   const failure = [8].indexOf(result) !== -1 ? true : false;
   const cancelled = [32].indexOf(result) !== -1 ? true : false;
   return (
     <div className="build-status">
-      <p>Commit</p>
-      <p>{commitId}</p>
-      {success && <button className="btn btn--success">Success</button>}
-      {failure && <button className="btn btn--fail">Failed</button>}
-      {cancelled && <button className="btn btn--warn">cancelled</button>}
+      Commit
+      <div>{commitId}</div>
+      {success && <button className="btn btn--success"><a href={commitBuildHref}>Success</a></button>}
+      {failure && <button className="btn btn--fail"><a href={commitBuildHref}>Failed</a></button>}
+      {cancelled && <button className="btn btn--warn"><a href={commitBuildHref}>Cancelled</a></button>}
```

Comment:
we can just add a class button on link elements instead : 
`<a class="button" href={commitBuildHref}>Cancelled</a>` instead of `<button className="btn btn--warn"><a href={commitBuildHref}>Cancelled</a></button>`